css/css-grid/parsing/grid-shorthand-serialization.html

```diff
@@ -20,6 +20,14 @@
     assert_equals(element.style.gridTemplateAreas, "none");
   }, "Serialization without grid-template-areas");
 
+  test(() => {
+    const gridTemplateAreasValue = '"one"';
+    element.style.grid = 'none'
+    element.style.gridTemplateAreas = gridTemplateAreasValue;
+    assert_equals(element.style.grid, "", "grid shorthand must not be serialized when grid-template-areas is set to a non-initial value");
```

Comment:
Can you check `element.style.gridTemplate` too?

Reply:
In this test or in a similar test in `/css/css-grid/parsing/grid-template-shorthand.html`?

Reply:
Strictly speaking grid-template-shorthand.html may be more suitable, but a oneliner here may be simpler and some tests below are already checking `gridTemplate`. Up to you